Lib/doctest.py

```diff
@@ -235,25 +237,17 @@ def _normalize_module(module, depth=2):
     else:
         raise TypeError("Expected a module, string, or None")
 
-def _newline_convert(data):
-    # The IO module provides a handy decoder for universal newline conversion
-    return IncrementalNewlineDecoder(None, True).decode(data, True)
-
 def _load_testfile(filename, package, module_relative, encoding):
     if module_relative:
         package = _normalize_module(package, 3)
+        with contextlib.suppress(Exception):
+            return (
+                importlib.resources.read_text(package, filename, encoding=encoding),
+                filename,
+            )
```

Comment:
Style nitpick. I think that the following code may look better:
```suggestion
            text = importlib.resources.read_text(package, filename,
                                                 encoding=encoding)
            return text, filename
```